plugins/home/src/homePageComponents/HeaderWorldClock/HeaderWorldClock.tsx

```diff
@@ -65,8 +66,14 @@ function getTimes(
       label = 'GMT';
     }
 
-    const time = d.toLocaleTimeString(lang, options);
-    clocks.push({ time, label });
+    const value = d.toLocaleTimeString(lang, options);
+    const dateTime = d.toLocaleTimeString(lang, {
+      timeZone: options.timeZone,
+      hour: '2-digit',
+      minute: '2-digit',
+      hour12: false,
+    });
```

Comment:
was this possible to configure before through options? just thinking we might want to keep that?

```suggestion
    const dateTime = d.toLocaleTimeString(lang, options);
```

or does the date time attribute expect specific values of hour, minute and hour12?

Reply:
The datetime only accepts [these](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/time# valid_datetime_values) valid values, we can't use the same options passed via props because the user can set an invalid format.

Reply:
got it - so currently the component accepts a `customTimeFormat` property - which means that we would have to stop support that prop if we want to make this change - is that correct?

Reply:
No, the displayed value still uses the user's format(the custom time format); the `datetime` is only consumed by the machine (enabling the screen reader, you will hear the displayed time).

Reply:
A-haaaaaa thats why you separated value and datetime ⭐  Thanks for explaining 🥇